Javascript/Activities.js

```diff
@@ -3,39 +3,51 @@ const mainContainer = document.querySelector('#activities-container');
     img: 'Images/japanese-food.jpg',
+    alt: 'cuisine',
   },
   {
     activity: 'Writing',
-    type: 'Kanji, Hiragana, Katakana, Romaji',
-    description: 'The Japanese script derives from the Chinese script, which was brought to Japan via Korea in the fourth century.',
+    type: '漢字, ひらがな, カタカナ',
+    description:
+      'The Japanese script derives from the Chinese script, which was brought to Japan via Korea in the fourth century.',
     img: 'Images/japanese-letter.jpg',
+    alt: 'writing',
   },
   {
     activity: 'Outfit',
-    type: 'Yukata, Kimono, Junihitoe, Keikogi, Jinbe',
-    description: 'Daily clothing in Japan is like typical Western clothing, pants and shirts are not new. Some outstanding clothing of traditional clothing can be said to be common in daily life.',
+    type: '服装',
+    description:
+      '<span class="text-hide">Daily clothing in Japan is like typical Western clothing, pants and shirts are not new.</span> Some outstanding clothing of traditional clothing can be said to be common in daily life.',
     img: 'Images/japanese-clothes.jpg',
+    alt: 'outfit',
   },
   {
     activity: 'Tea Ceremony',
     type: '抹茶, 茶の湯, 茶事',
-    description: 'The Japanese tea ceremony is a ritual way of preparing green or matcha tea, influenced by Zen Buddhism, served to a small group of guests in a tranquil setting.',
+    description:
+      'The Japanese tea ceremony is a ritual way of preparing green or matcha tea. <span class="text-hide">Influenced by Zen Buddhism, served to a small group of guests in a tranquil setting.</span>',
     img: 'Images/japanese-tea.jpg',
+    alt: 'tea',
   },
   {
     activity: 'Architecture',
     type: '日本建築',
-    description: 'Mostly, the buildings that have survived to this day and that show characteristics of traditional Japanese architecture are castles, Buddhist temples, and Shinto shrines.',
+    description:
+      '<span class="text-hide">Mostly, the buildings that have survived to this day and that show characteristics of traditional</span> Japanese architecture are castles, Buddhist temples, and Shinto shrines.',
     img: 'Images/japanese-buildings.jpg',
+    alt: 'architecture',
   },
   {
     activity: 'Hanami',
     type: '花見',
-    description: 'Hanami is the Japanese tradition of observing the beauty of flowers, but this word is generally associated with the period when the cherry blossoms are in bloom and when the Japanese flock to parks and gardens to gaze at their blossoms (sakura).',
+    description:
+      'Hanami is the Japanese tradition of observing the beauty of flowers<span class="text-hide">, but this word is generally associated with the period when the cherry blossoms are in bloom and when the Japanese flock to parks and gardens to gaze at their blossoms (sakura).</span>',
     img: 'Images/japanese-hanami.jpg',
+    alt: 'hanami',
```

Comment:
Weldone in implementing the activities section using the dynamic javascript feature, Please in the design given there is a background image attached on each of them. Kindly implement this feature so looks close to the design given 😄 

Design given :
![image](https://github.com/Luffytaro22/Capstone-Japan/assets/20537712/8794163e-5d41-459c-8f66-36c2bfacb5f7)

Your design :
![image](https://github.com/Luffytaro22/Capstone-Japan/assets/20537712/15100174-8cae-4cb0-8753-089267191307)